toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -34,6 +35,8 @@ func doCustomizations(buildDir string, baseConfigPath string, config *imagecusto
 	// Note: The ordering of the customization steps here should try to mirror the order of the equivalent steps in imager
 	// tool as closely as possible.
 
+	buildTime := time.Now().Format("2006-01-02T15:04:05Z")
```

Comment:
nit: Only retrieve `time.Now()` here. Let `addCustomizerRelease` do the string formatting.